yellowpages.lic

```diff
@@ -0,0 +1,56 @@
+#Load base contacts YAML
+Contacts = YAML.load_file('./scripts/data/base-contacts.yaml')
+Contacts = Contacts['contacts']
+
+#DEBUG INFO
+#puts "---Base Contacts---"
+#puts Contacts.inspect
+#puts Contacts.class
+
+#Check for <CharName>-contacts.yaml file and merges it if exists
+if File.exist?('./scripts/profiles/'+checkname+'-contacts.yaml')
+	puts "Personal Contact file found! Now importing!"

```

Comment:
I don't think `puts` will output on all Lich front-ends. Our convention has been to use `echo` instead.